static/js/import_wizard2/components/mapping_templates/multi_var_mulit_date_col.tsx

```diff
@@ -34,106 +33,44 @@ import { MappingHeaderInput } from "../shared/mapping_header_input";
-                }
-                orderedColumns={props.csvData.orderedColumns}
-              />
-            </td>
-          </tr>
-          <tr>
-            <td>Observation Value*:</td>
-            <td>
-              <MappingHeaderInput
-                mappedThingName={
-                  MAPPED_THING_NAMES[MappedThing.DATE] || MappedThing.DATE
-                }
-                mappingVal={props.userMapping.get(MappedThing.DATE)}
-                onMappingValUpdate={(mappingVal: MappingVal) =>
-                  onMappingValUpdate(MappedThing.DATE, mappingVal)
-                }
-                orderedColumns={props.csvData.orderedColumns}
-              />
-            </td>
-          </tr>
-          <tr>
-            <td>Unit:</td>
-            <td>
-              <MappingColumnInput
-                mappingVal={props.userMapping.get(MappedThing.UNIT)}
-                onMappingValUpdate={(mappingVal) =>
-                  onMappingValUpdate(MappedThing.UNIT, mappingVal)
-                }
-                orderedColumns={props.csvData.orderedColumns}
-              />
-            </td>
-          </tr>
-          <tr>
-            <td>Ignored columns:</td>
-            <td>
-              {props.csvData.orderedColumns.map((col: Column) => {
-                if (mappedColumnIndices.has(col.columnIdx)) {
-                  return "";
-                }
-                return col.header;
-              })}
-            </td>
-          </tr>
-        </tbody>
-      </table>
+    <div id="multi-var-multi-date">
+      <MappingColumnInput
+        mappedThing={MappedThing.STAT_VAR}
+        mappingVal={props.userMapping.get(MappedThing.STAT_VAR)}
+        onMappingValUpdate={(mappingVal) =>
+          props.onMappingValUpdated(MappedThing.STAT_VAR, mappingVal)
+        }
+        orderedColumns={props.csvData.orderedColumns}
+        isRequired={true}
+      />
+      <MappingPlaceInput
+        mappingType={MappingType.COLUMN}
+        mappingVal={props.userMapping.get(MappedThing.PLACE)}
+        onMappingValUpdate={(mappingVal: MappingVal) =>
+          props.onMappingValUpdated(MappedThing.PLACE, mappingVal)
+        }
+        orderedColumns={props.csvData.orderedColumns}
+      />
+      <MappingHeaderInput
+        mappedThingName={
+          MAPPED_THING_NAMES[MappedThing.DATE] || MappedThing.DATE
+        }
+        mappingVal={props.userMapping.get(MappedThing.DATE)}
+        onMappingValUpdate={(mappingVal: MappingVal) =>
+          props.onMappingValUpdated(MappedThing.DATE, mappingVal)
+        }
+        orderedColumns={props.csvData.orderedColumns}
+      />
+      <MappingColumnInput
+        mappedThing={MappedThing.UNIT}
+        mappingVal={props.userMapping.get(MappedThing.UNIT)}
+        onMappingValUpdate={(mappingVal) =>
+          props.onMappingValUpdated(MappedThing.UNIT, mappingVal)
+        }
+        orderedColumns={props.csvData.orderedColumns}
+        isRequired={false}
+      />
```

Comment:
These 4 components pattern is repeated too. Can you have another component to wrap and parameterize them?

Reply:
from offline discussion, since different templates will use a different combination of inputs, will keep this as is